src/simplewallet/simplewallet.cpp

```diff
@@ -196,7 +196,7 @@ namespace
                             "  account tag_description <tag_name> <description>");
   const char* USAGE_ADDRESS("address [ new <label text with white spaces allowed> | all | <index_min> [<index_max>] | label <index> <label text with white spaces allowed> | device [<index>]]");
   const char* USAGE_INTEGRATED_ADDRESS("integrated_address [device] [<payment_id> | <address>]");
-  const char* USAGE_ADDRESS_BOOK("address_book [(add ((<address> [pid <id>])|<integrated address>) [<description possibly with whitespaces>])|(delete <index>)]");
+  const char* USAGE_ADDRESS_BOOK("address_book [(add ((<address>)|<integrated address>) [<description possibly with whitespaces>])|(delete <index>)]");
```

Comment:
Should `((<address>)|<integrated address>)` be `(<address>|<integrated address>)`?